core-bundle/tests/Functional/Util/SimpleToken/StringExtensionProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Functional\Util\SimpleToken;
+
+use Symfony\Component\ExpressionLanguage\ExpressionFunction;
+use Symfony\Component\ExpressionLanguage\ExpressionFunctionProviderInterface;
+
+class StringExtensionProvider implements ExpressionFunctionProviderInterface
+{
+    public function getFunctions(): array
+    {
+        return [
+            ExpressionFunction::fromPhp('strtoupper'),
```

Comment:
What do we need this for?

Reply:
It's a custom extension for the simple token expression language (this being a fixture for the related functional test).

Reply:
This is just a functional test, that's correct :)

Reply:
Should this go in the `Fixtures` dir?